app/Http/Controllers/Api/TagsController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Rogue\Http\Controllers\Api;
+
+use Rogue\Http\Requests\TagsRequest;
+use Rogue\Repositories\PostRepository;
+use Rogue\Http\Transformers\PostTransformer;
+
+class TagsController extends ApiController
+{
+    /**
+     * The post service instance.
+     *
+     * @var Rogue\Repositories\PostRepository
+     */
+    protected $post;
+
+    /**
+     * @var \League\Fractal\TransformerAbstract;
+     */
+    protected $transformer;
+
+    /**
+     * Create a controller instance.
+     *
+     * @param  PostContract $posts
+     * @return void
+     */
+    public function __construct(PostRepository $post)
+    {
+        $this->post = $post;
+        $this->transformer = new PostTransformer;
+    }
+
+    public function store(TagsRequest $request)
```

Comment:
I'm a little concerned we'll run into tiny differences we'd want to implement between `store` methods in different controllers, which we couldn't easily override with a trait. I've generally found traits are more useful for tinier bits of optional functionality.